ui/site/css/streamer/_header.scss

```diff
@@ -28,8 +27,6 @@
     @extend %roboto;
 
     font-size: 2.2rem;
-    text-transform: uppercase;
-    letter-spacing: 4px;
```

Comment:
This is unrelated but seemed weirdly inconsistent to me. On the streamer list page, the names are just shown normally and imo it looks better that way.